src/utils.rs

```diff
@@ -265,10 +265,20 @@ pub fn validate_contract_deployed<S: StateReader + Clone>(
     get_deployed_address_class_hash_at_address(state, contract_address)
 }
 
+pub fn calculate_sn_keccak(data: &[u8]) -> [u8; 32] {
+    let mut hasher = Keccak256::default();
+    hasher.update(data);
+    let mut result: [u8; 32] = hasher.finalize().try_into().unwrap();
+    // Only the first 250 bits from the hash are used.
+    result[0] &= 0b0000_0011;
+    result
+}
+
 //* -------------------
 //*      Macros
 //* -------------------
 
+use sha3::{Digest, Keccak256};
```

Comment:
Shouldn't those `use` statements be at the top?

Reply:
true, moved